### ISSUE TITLE
Division by zero

### ISSUE THREAD

**Issue by reox** [nextcloud/recognize#439]
**Describe the bug**
Got this in the log 4880 times since yesterday (When I upgraded to 25.0.0 and installed recognize):
```
Error: Division by zero at /var/www/nextcloud/apps/recognize/lib/Service/FaceClusterAnalyzer.php#190

    /var/www/nextcloud/apps/recognize/lib/Service/FaceClusterAnalyzer.php - line 190:

    OC\Log\ErrorHandler::onError()

    <<closure>>

    OCA\Recognize\Service\FaceClusterAnalyzer::OCA\Recognize\Service\{closure}("*** sensiti ... *")

    /var/www/nextcloud/apps/recognize/lib/Service/FaceClusterAnalyzer.php - line 189:

    array_map()

    /var/www/nextcloud/apps/recognize/lib/Service/FaceClusterAnalyzer.php - line 90:

    OCA\Recognize\Service\FaceClusterAnalyzer::calculateCentroidOfDetections()

    /var/www/nextcloud/apps/recognize/lib/BackgroundJobs/ClusterFacesJob.php - line 28:

    OCA\Recognize\Service\FaceClusterAnalyzer->calculateClusters()

    /var/www/nextcloud/lib/public/BackgroundJob/Job.php - line 78:

    OCA\Recognize\BackgroundJobs\ClusterFacesJob->run()

    /var/www/nextcloud/lib/public/BackgroundJob/QueuedJob.php - line 58:

    OCP\BackgroundJob\Job->start()

    /var/www/nextcloud/lib/public/BackgroundJob/QueuedJob.php - line 48:

    OCP\BackgroundJob\QueuedJob->start()

    /var/www/nextcloud/cron.php - line 152:

    OCP\BackgroundJob\QueuedJob->execute()
```

**To Reproduce**
1. Run the background job I guess?

**Expected behavior**
No division by zero

**Recognize (please complete the following information):**
 - JS-only mode: No
 - Enabled modes: Face, Object, Landmarks
 - Version 3.1.0

**Server (please complete the following information):**
 - Nextcloud: 25.0.0
 - OS: Debian
 - RAM: 6GB
 - Processor Architecture: x86-64

**Additional context**

Was that already fixed by https://github.com/nextcloud/recognize/commit/6b236ce0d0531e5172d6af35d18194fae3fa5355 ?

**Comment by reox** [comment]
ah I see, this is a duplicate of #397
I swear I searched the issues before but that one did not show up... probably a typo while searching 😅

**Comment by marcelklehr** [comment]
> Was that already fixed by https://github.com/nextcloud/recognize/commit/6b236ce0d0531e5172d6af35d18194fae3fa5355 ?

Yes, should be fixed with v3.1.1
